css/style.css

```diff
@@ -0,0 +1,442 @@
+body {
+	font-family: 'GothamPro';
+	max-width: 1030px;
+	margin: 0 auto;
+}
+.header__menu {
+	height: 115px;
+	display: flex;
+	align-items: center;
+}
+.header__logo {
+    width: 55px;
+    height: 55px;
+    display: flex;
+    align-items: center;
+    flex: 0.1;
+    margin-right: 18px;
+}
+.menu {
+	flex: 1.9;
+	display: flex;
+	justify-content: space-around;
+}
+.menu__item {
+	color: #000;
+	text-decoration: none;
+	font-size: 12px;
+	font-weight: bold;
+	display: flex;
+}
+.menu__item:hover {
+	color: #aaa;
+}
+.menu__profile {
+	flex: 1;
+	display: flex;
+	justify-content: space-around;
+}
+.menu__USA {
+	flex: 0.1;
+	display: flex;
+	justify-content: flex-end;
+}
+.header__logo_mobile {
+	display: none;
+}
+.menu__burger {
+	display: none;
+}
+.menu__form {
+	width: 178px;
+	flex: 1.7;
+	display: flex;
+	justify-content: flex-end;
+}
+.menu__input {
+	height: 30px;
+	border-top: 0px;
+	border-right: 0px;
+	border-left: 0px;
+	border-bottom: 1px solid #000;
+	position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5980773)